introduction_to_amazon_algorithms/ntm_synthetic/ntm_synthetic.ipynb

```diff
@@ -65,13 +65,14 @@
    },
    "outputs": [],
    "source": [
-    "bucket = '<your_s3_bucket_name_here>'\n",
-    "prefix = 'sagemaker/DEMO-ntm-synthetic'\n",
-    " \n",
-    "# Define IAM role\n",
-    "import boto3\n",
-    "import re\n",
-    "from sagemaker import get_execution_role\n",
+   "# Define IAM role\n",
+   "import sagemaker\n",
+   "import boto3\n",
+   "import re\n",
+   "from sagemaker import get_execution_role\n",
+   "\n",
+   "sess = sagemaker.Session()\n",
+   "bucket=sess.default_bucket()"",
```

Comment:
There are two quotation marks at the end of this line.

Reply:
Fixed